tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -112,6 +112,7 @@ def tap_comment(self):
         element.tap()
         self.marionette.execute_script(
             'arguments[0].scrollIntoView(true);', [element])
+        Wait(self.marionette, timeout=120).until(expected.element_displayed(*self._comment_locator))
         element = self.marionette.find_element(*self._comment_locator)
```

Comment:
I agree. element_displayed() takes the element as well. 
I also suggest moving line 116 above Wait() call, and perhaps using a different variable name for style (since 'element' name is already used above), such as comment =  self.marionette.find_element(*self._comment_locator)
Wait(self.marionette, timeout=120).until(expected.element_displayed(comment))